scripts/general.lua

```diff
@@ -79,4 +79,21 @@ events.TICK:register(function ()
 	end
 end)
 
+events.RENDER:register(function (delta, context)
+    local shouldRenderHead = not renderer:isFirstPerson()
+    if context == "FIRST_PERSON" then
+        shouldRenderHead = false
+    end
+    if context == "OTHER" then
+        shouldRenderHead = true
+    end
+    if context == "PAPERDOLL" then
+        shouldRenderHead = true
+    end
+    if context == "MINECRAFT_GUI" then
+        shouldRenderHead = true
+    end
+    models.models.main.Avatar.Head:setVisible(shouldRenderHead)
```

Comment:
In these code, I think that you want to hide the head model in first person perspective with First Person Model or Real Camera. However, these code doesn't seem to work. Even if I checked with only Figura and First Person Model installed. Can you fix this?

![2024-03-04_22 58 09](https://github.com/Gakuto1112/SenkoSan/assets/90630001/d7ef1618-6939-4816-826f-8de5bf705ae7)

Reply:
I retried it and it didn't have any problems

Reply:
If so, you may need to reset your model and re-upload it first, otherwise it will be the old code model that is used

Reply:
> Just a quick question, have you uploaded your model to Figura's servers before?

I tested the avatar with  your changes.
This is tested with the local avatar. (Actually, my global avatar is another one.) I think that this problem doesn't matter if the avatar is local or global.

Reply:
I resolved this problem by hiding the head in `OTHER` context.

```lua
events.RENDER:register(function (_, context)
	models.models.main.Avatar.Head:setVisible(context ~= "OTHER")
end)
```

However, the shadow of the head isn't casted. This doesn't look good...

![2024-03-05_11 24 09](https://github.com/Gakuto1112/SenkoSan/assets/90630001/99df8785-4bd5-451c-9f3d-6622ff2ff458)

Reply:
> I resolved this problem by hiding the head in `OTHER` context.
> 
> ```lua
> events.RENDER:register(function (_, context)
> 	models.models.main.Avatar.Head:setVisible(context ~= "OTHER")
> end)
> ```
> 
> However, the shadow of the head isn't casted. This doesn't look good...
> 
> ![2024-03-05_11 24 09](https://private-user-images.githubusercontent.com/90630001/309961536-99df8785-4bd5-451c-9f3d-6622ff2ff458.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2MDY2NTcsIm5iZiI6MTcwOTYwNjM1NywicGF0aCI6Ii85MDYzMDAwMS8zMDk5NjE1MzYtOTlkZjg3ODUtNGJkNS00NTFjLTlmM2QtNjYyMmZmMmZmNDU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA1VDAyMzkxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2YTAxYTI5ODQ4ZjA0NWJjZjU1ODIyZjlhMmE1ZDM2NTcwOGNkNjg0OTg5MWUzNDA5ZmZiMzlmN2M5NDBlNTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AJ_JLEkKyHk5AQdu5SXB50GaE17Iopibf3ukialaCqY)

I resolved this problem by making the head transparent instead of hiding. Can you try it with the below code? If there is no problem, please update your pull request.

```lua
events.RENDER:register(function (_, context)
    models.models.main.Avatar.Head:setOpacity(context ~= "OTHER" and 1 or 0)
end)
```

![2024-03-05_11 59 48](https://github.com/Gakuto1112/SenkoSan/assets/90630001/0e642a5d-c94f-4b19-95f6-6382e1f1153d)

Reply:
I resolved this problem by hiding the head in `OTHER` context.

```lua
events.RENDER:register(function (_, context)
	models.models.main.Avatar.Head:setVisible(context ~= "OTHER")
end)
```

However, the shadow of the head isn't casted. This doesn't look good...

![2024-03-05_11 24 09](https://github.com/Gakuto1112/SenkoSan/assets/90630001/99df8785-4bd5-451c-9f3d-6622ff2ff458)